core/src/main/java/com/capitalone/dashboard/model/Issue.java

```diff
@@ -0,0 +1,187 @@
+package com.capitalone.dashboard.model;
+
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+/**
+ * A specific commit in a version control repository.
+ *
+ * Possible collectors:
+ *  Subversion (in scope)
+ *  Git (in scope)
+ *  GitHub
+ *  TFS
+ *  BitBucket
+ *  Unfuddle
+ *
+ */
+@Document(collection="issues")
+public class Issue extends SCM {
+    @Id
+    private ObjectId id;
+    private ObjectId collectorItemId;
+
+    //Thus could be user id in GIT
+    private String orgName;
+    private String repoName;
+    private String number;
+    private String userId = null;
+    private String createdAt;
+    private String closedAt;
+    private long timestamp;
+    private long resolutiontime;
+
+    //These fields are for another collector to fill later - the ones
```

Comment:
Collectors are meant to be independent of each-other. What does this comment mean?